_data/developer.yml

```
@@ -60,6 +60,15 @@ websites:
     doc: https://confluence.atlassian.com/cloud/secure-your-account-with-two-step-verification-939505063.html
     exception: "2FA settings are service-specific."
 
+  - name: Backlog
+    url: https://backlog.com
+    img: backlog.png
+    tfa:
+      - u2f
+      - totp
+      - sms
+    doc: https://support.backlog.com/hc/en-us/community/posts/360029520833
```

Old link doesn't work for me. Here's one that does work:
```suggestion
    doc: https://support.nulab.com/hc/en-us/articles/115001025087
```